modules/gapi/include/opencv2/gapi/gmat.hpp

```diff
@@ -213,18 +206,12 @@ struct GAPI_EXPORTS GMatDesc
 static inline GMatDesc empty_gmat_desc() { return GMatDesc{-1,-1,{-1,-1}}; }
 
 #if !defined(GAPI_STANDALONE)
-class Mat;
-GAPI_EXPORTS GMatDesc descr_of(const cv::Mat &mat);
 GAPI_EXPORTS GMatDesc descr_of(const cv::UMat &mat);
 #endif // !defined(GAPI_STANDALONE)
 
+GAPI_EXPORTS GMatDesc descr_of(const cv::Mat &mat);
 /** @} */
 
-// FIXME: WHY??? WHY it is under different namespace?
-namespace gapi { namespace own {
-    GAPI_EXPORTS GMatDesc descr_of(const Mat &mat);
-}}//gapi::own
```

Comment:
So does `own::Mat` has its own overload now in the OpenCV build or not? How can we test the `own::Mat` in our tests now? (I mean, the `own::Mat`'s own tests)

Reply:
Tests and cv::gapi::own::descr_of returned. Fixed.